src/compose/ComposeMenu.js

```diff
@@ -35,7 +28,7 @@ type SelectorProps = $ReadOnly<{||}>;
 
 type Props = $ReadOnly<{|
   ...OuterProps,
-
```

Comment:
Please keep this blank line, for separation, and add a new one below `showActionSheetWithOptions`.